### PR TITLE
PYTHON-4604 Bundle libmongocrypt 1.11.0

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: bindings/python/sbom.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
   ],
   "dependencies": [
     {
-      "ref": "pkg:github/mongodb/libmongocrypt@1.10.1"
+      "ref": "pkg:github/mongodb/libmongocrypt@1.11.0"
     }
   ],
   "metadata": {
-    "timestamp": "2024-06-28T19:11:17.100829+00:00",
+    "timestamp": "2024-08-08T02:35:36.986098+00:00",
```
Comment (NoahStapp) [inline bindings/python/sbom.json:28]: Is this always going to be hardcoded?

**Review by NoahStapp — APPROVED** [review]
One clarifying question, otherwise LGTM!

File: bindings/python/sbom.json (hunk 1 above)
Comment (blink1073) [reply bindings/python/sbom.json:28]: The file is auto-generated by Silk

**Comment by ShaneHarvey** [comment]
Do we need to do any work in pymongocrypt for PYTHON-4392 before we release this?

**Comment by blink1073** [comment]
> Do we need to do any work in pymongocrypt for [PYTHON-4392](https://jira.mongodb.org/browse/PYTHON-4392) before we release this?

The other way around, we need a release of pymongocrypt to support PYTHON-4392

**Comment by ShaneHarvey** [comment]
Right, I get that but I'm wondering if we need to make any changes to the bindings besides upgrading to 1.11+.

**Comment by blink1073** [comment]
> Right, I get that but I'm wondering if we need to make any changes to the bindings besides upgrading to 1.11+.

I updated the bindings, and there were only docstring changes

**Comment by blink1073** [comment]
@ShaneHarvey good to merge?

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**